src/ngraph/runtime/backend.hpp

```diff
@@ -124,6 +124,16 @@ class ngraph::runtime::Backend
     /// \returns true if the op is supported, false otherwise.
     virtual bool is_supported(const Node& node) const;
 
+    /// \brief A set of properties supported by a backend
+    enum class Property
+    {
+        memory_attach /// A tensor can attach the user memory
```

Comment:
its not clear what this means. Could you perhaps explain in more detail?

Reply:
@jbobba I explained these things in PPT I sent you some time ago. We discussed that.
or maybe you mean a comment in the code is not clear?

Reply:
i dont recall the PPT but would be nice to make the comment here more self-explanatory.